src/app/registration/registration.component.html

```diff
@@ -1,40 +1,141 @@
+                  </div>
+                </div>
+                </a>
+                <div class="text-white fs-16 mt-2 min-w-100">Login / SignUp</div>
+              </li>
+              <li role="presentation"><a href="#optimization" aria-controls="optimization" role="tab" data-toggle="tab">
+                <div class="orange-border">
+                  <div class="white-box">
+                    <div class="i" aria-hidden="true"> 3</div>
+                  </div>
+                </div>
+                </a>
+                <div class="text-white fs-16 mt-2 min-w-100">Registration Form</div>
+              </li>
+              
+             
+            </ul>
+            <!-- end design process steps--> 
+            <!-- Tab panes -->
+            <div class="tab-content mt-5">
+              <div role="tabpanel" class="tab-pane active" id="discover">
+                <div class="fs-24 text-white font-600">Please loginto AVSAR with your Meri-Pehchaan credentials</div>
+                <div class="fs-18 pt-4 text-white font-400">
+                  1. If you have digilocker login with Pin enter your phone number on mobile tab and pin or username and pin on username tab or aadhaar and pin in others tab.
+                </div>
+                <div class="fs-18 pt-4 text-white font-400">
+                  2. If you dont remember your digilocker pin, then download digilocker and create account and set pin and login as per instruction 1.
+                </div>
+                <div class="mt-5 mb-5">
+                  <div class="iframe-container1">
+                    <iframe class="responsive-iframe" width="582px" height="276px" src="https://www.youtube.com/embed/D0UnqGm_miA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                  </div>
+                </div>
+              </div>
+              <div role="tabpanel" class="tab-pane" id="strategy">
+                <div class="mt-5 mb-5">
+                  <div class="iframe-container1">
+                    <iframe class="responsive-iframe" width="582px" height="276px" src="https://www.youtube.com/embed/D0UnqGm_miA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                  </div>
+                </div>
+              </div>
+              <div role="tabpanel" class="tab-pane" id="optimization">
+                <div class="mt-5 mb-5">
+                  <div class="iframe-container1">
+                    <iframe class="responsive-iframe" width="582px" height="276px" src="https://www.youtube.com/embed/D0UnqGm_miA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
@shreyas1434shinde we need to make this url configurable.

Reply:
@itsvick yes it's youtube iframe it is configurable